eng/scripts/automation/GenerateAndBuildLib.ps1

```diff
@@ -223,58 +272,56 @@ function New-MgmtPackageFolder() {
         [string]$service = "",
         [string]$packageName = "",
         [string]$sdkPath = "",
-        [string]$commitid = "",
         [string]$readme = "",
         [string]$AUTOREST_CONFIG_FILE = "autorest.md",
         [string]$outputJsonFile = "newPacakgeOutput.json"
     )
   
-    $projectFolder="$sdkPath/sdk/$packageName/Azure.ResourceManager.*"
+    if ($packageName -eq "") {
+        $packageName = $service
+    }
+
+    $projectFolder = (Join-Path $sdkPath "sdk" $packageName "Azure.ResourceManager.*")
```

Comment:
Do we need to resolve the `*`?

Reply:
Yes. for mgmt, most case the package name is `Azure.ResourceManager.<serviceDirectory>`, but it may have exception in current already released RPs. So we use this `*` to resolve out the exact existing package name.